src/components/organisms/DropdownContent/DropdownContent.tsx

```diff
@@ -0,0 +1,150 @@
+  onChange,
+  setIsOpen,
+  helperText,
+  selectedOptionObjects,
+  tags,
+}) => {
+  const [selectedValue, setSelectedValue] = useState<string[]>(
+    value ? (Array.isArray(value) ? value : [value]) : []
+  )
+
+  const handleSingleSelect = (selectedOption: string) => {
+    onChange(selectedOption ? selectedOption : '')
+  }
+
+  const handleMultipleSelect = (selectedOption: string) => {
+    const selectedValues = Array.isArray(selectedValue)
+      ? [...selectedValue]
+      : []
+    const optionIndex = selectedValues.indexOf(selectedOption)
+
+    const newSelectedValues =
+      optionIndex === -1
+        ? [...selectedValues, selectedOption]
+        : selectedValues.filter((value) => value !== selectedOption)
+
+    setSelectedValue(newSelectedValues)
+
+    if (setIsOpen) {
+      setIsOpen(true)
+    }
+  }
+
+  const handleOnSave = () => {
+    onChange(selectedValue)
+  }
+
+  const handleCancel = () => {
+    setSelectedValue([])
+  }
+
+  return (
+    <>
+      <div
+        className={classNames(classes.dropdownMenu, classes[dropdownSize])}
+        hidden={disabled || !isOpen}
+      >
+        <div hidden={!searchInput}>{searchInput}</div>
+
+        <ul>
+          {map(options, (option, index) => {
+            const isSelected =
+              selectedValue && includes(selectedValue, option?.value)
+
+            return (
+              <li key={index} className={classes.dropdownMenuItem}>
+                {multiple && (
+                  <CheckBoxInput
+                    name={name}
+                    ariaLabel={ariaLabel}
+                    label={option?.label}
+                    value={isSelected || false}
+                    className={classes.option}
+                    onChange={() => handleMultipleSelect(option?.value)}
+                  />
+                )}
+                <p
+                  className={classNames(
+                    classes.option,
+                    isSelected && classes.selectedOption
+                  )}
+                  hidden={multiple}
+                  onClick={() => handleSingleSelect(option?.value)}
+                >
+                  {option?.label}
+                </p>
+              </li>
+            )
+          })}
+          <div
+            hidden={!buttons}
+            className={classNames(buttons && classes.buttonsContainer)}
+          >
+            <Button
+              appearance={AppearanceTypes.Secondary}
+              size={SizeTypes.S}
+              onClick={handleCancel}
+            >
+              {cancelButtonLabel}
+            </Button>
+            <Button
+              appearance={AppearanceTypes.Primary}
+              size={SizeTypes.S}
+              onClick={handleOnSave}
+              className={classes.dropdownButton}
+            >
+              {proceedButtonLabel}
+            </Button>
+          </div>
+        </ul>
+      </div>
+
+      <p hidden={!helperText} className={classes.helperText}>
```

Comment:
**[blocker]**

Noticed that this helpertext ruins the positioning of error. Error should be under the input field itself

 
<img width="573" alt="Screenshot 2023-06-01 at 08 34 05" src="https://github.com/keeleinstituut/tv-web/assets/26221722/e39f301c-e8ab-4ce3-a78c-9639588b8a9b">

Reply:
Extra note -- I did some minor improvements to the error message + the testing form we have in Components page, would suggest pulling those changes in as well